### ISSUE TITLE
Manually create exclusions by hand

### ISSUE THREAD

**Issue by albotroz** [d4rken-org/sdmaid-se#139]
It should be possible to manually add Apps, folders and individual files into the Exclusion Manager option/ window via the Exclusion Manager and not only after performing scans.

**Comment by Shkzeen** [comment]
I had in idea, might not be a great one.. but it should do
why not pre-convert the values and save both of them.
l8r pals.

**Comment by d4rken** [comment]
FYI little example i typed up for #317:

> That will also be possible with #139 you just have to make sure to stay a little vague, i.e. create `Android/data/XXXXX` not `/data/media/0/Android/data/XXXXX` because internally the paths can be different depending on what access method is used.
> 
> Example:
> 
> * Normal: `/storage/4BBD-D3E7/Android/data/some.app/specialfolder`
> * SAF: `content://com.android.externalstorage.documents/tree/4BBD-D3E7/Android/data/some.app/specialfolder`
> * Root: `/data/media/0/Android/data/some.app/specialfolder`
> 
> It's all the same folder, just different way of access which is why creating a "file picker" to create exclusion is complicated.
> 
> But internally SD Maid converts all paths into "Segments", so the above three paths internally look like:
> 
> * Normal: Base `/storage/4BBD-D3E7`, Segments `[Android, data, some.app, specialfolder]`
> * SAF: Base `content://com.android.externalstorage.documents/tree/4BBD-D3E7/`, Segments `[Android, data, some.app, specialfolder]`
> * Root: Base `/data/media/0/`, Segments `[Android, data, some.app, specialfolder]`
> 
> So an exclusion based on the later segments, i.e. `[Android, data, some.app, specialfolder]` would work as exclusion for all access types, and would be fast too, less conversion to do for SD Maid.
> 
> Closing in favor of #139
